ppr-ui/src/composables/mhrRegistration/useNewMhrRegistration.ts

```diff
@@ -275,14 +276,16 @@ export const useNewMhrRegistration = () => {
 
   const buildApiData = (): NewMhrRegistrationApiIF => {
     const data: NewMhrRegistrationApiIF = {
-      documentId: getMhrRegistrationDocumentId.value,
       ownLand: getMhrRegistrationOwnLand.value,
       submittingParty: parseSubmittingParty(),
       ownerGroups: parseOwnerGroups(),
       location: parseLocation(),
       description: parseDescription(),
       ...(isRoleStaffReg.value && !!getStaffPayment.value && {
         clientReferenceId: getStaffPayment.value.folioNumber
+      }),
+      ...(!isMhrManufactuerRegistration.value && {
```

Comment:
The `documentId` is not required when registering a Manufacturer? Is that right?

Reply:
Based on the ticket yes.

![image](https://github.com/bcgov/ppr/assets/77707952/d8eb311e-e17e-42d0-9220-88b8e774705d)

Reply:
Ok cool, and we are reusing the same IF for registering Manufacturers?

Reply:
At the moment yes, we can change that if there is a lot of differences in the payload.